struct.go

```diff
@@ -334,11 +355,23 @@ func (s *Section) reflectFrom(val reflect.Value) error {
 		tpField := typ.Field(i)
 
 		tag := tpField.Tag.Get("ini")
-		if tag == "-" {
+		opts := strings.Split(tag, ",")
+		if len(opts) > 2 {
+			return fmt.Errorf("Expected max. 2 comma-separated values in a tag, %d given", len(opts))
+		}
+
+		if opts[0] == "-" {
```

Comment:
If tag is `-`, then should get out of loop directly, so we need `if tag == "-" {`, then do the `omitempty` check.